packages/ui/src/components/input/NetworkSelect.tsx

```diff
@@ -159,6 +162,17 @@ const NetworkSelect: FunctionComponent<{
                                     }
                                 />
                             ))}
+                    <li
```

Comment:
we have a `LinkButton` component that can probably be used here

Reply:
I tried using it. It is mainly made for buttons like "Next", "Done", "Cancel" and so on. 
To use it in this context, I must override all the CSS using the classes prop.

Reply:
sorry I made a mistake on the intended component, the correct one would be `ClickableText`